subprojects/gradle/slack/src/integTest/kotlin/com/avito/slack/SlackConditionalSenderIntegrationTest.kt

```diff
@@ -6,16 +6,26 @@ import com.avito.slack.model.SlackSendMessageRequest
 import com.avito.utils.logging.CILogger
 import com.google.common.truth.Truth.assertThat
 import org.funktionale.tries.Try
+import org.junit.jupiter.api.Assumptions
 import org.junit.jupiter.api.Test
 import java.util.UUID
 
 internal class SlackConditionalSenderIntegrationTest {
 
-    //todo хреново что для локального запуска нужно здесь ENV задавать
-    private val testChannel = SlackChannel(requireNotNull(System.getenv("SLACK_TEST_CHANNEL")))
-    private val testToken = requireNotNull(System.getenv("SLACK_TEST_TOKEN"))
-    private val slackClient: SlackClient =
-        SlackClient.Impl(testToken, requireNotNull(System.getenv("SLACK_TEST_WORKSPACE")))
+    private val testChannel by lazy {
+        System.getenv("SLACK_TEST_CHANNEL").assumeNotNullOrBlank().let { SlackChannel(it) }
+    }
+
+    private val testToken by lazy {
+        System.getenv("SLACK_TEST_TOKEN").assumeNotNullOrBlank()
```

Comment:
i think we must mark it Depreceted and try to pass parameters via constructor

Reply:
Do you know another way to pass a value to a unit-test?

Reply:
Constructor. We can abstract env logic from this class becuase it is dependency details leak

Reply:
Constructor of what exactly? 
I don't see a difference. We need to get value for tests.

Reply:
Sorry i don' understand. Why do we need env for test?
I want hide token impl details.

Reply:
it's an integration test. we need slack hook token to execute it

Reply:
Can we get slack data not from environment?

Reply:
not much of a choice: either env or gradle param